indica/readers/read_st40.py

```diff
@@ -129,8 +129,8 @@ def get_raw_data(self, uid: str, instrument: str, revision: RevisionLike = 0):
                     continue
 
                 transform = data[quant].transform
-                if hasattr(transform, "set_equilibrium"):
-                    transform.set_equilibrium(self.equilibrium)
+                # if hasattr(transform, "set_equilibrium"):
```

Comment:
This is needed in a few workflows that don't have tests yet :-(.
I'll reinstate it with a keyword set to False by default so your stuff doesn't have to be changed.
We anyway have to refactor this wrapper in the next weeks.